x-pack/plugins/cases/common/api/metrics/case.ts

```diff
@@ -22,9 +22,9 @@ const StatusInfoRt = rt.type({
    */
   inProgressDuration: rt.number,
   /**
-   * The number of times the case was reopened after being closed.
+   * The ISO string representation of the dates the case was reopened
    */
-  numberOfReopens: rt.number,
+  reopenDates: rt.array(rt.string),
```

Comment:
Is formatting down to `ms` level really needed?

![image](https://user-images.githubusercontent.com/7405507/149519811-3c130e69-a951-470f-8aec-5b684901f3d3.png)

Reply:
Issue https://github.com/elastic/kibana/issues/123061 has been raised to address the granularity of timestamps separately.